src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php

```diff
@@ -175,17 +176,34 @@ private function generateItems(OrderInterface $order): void
     {
         $numberOfItems = random_int(1, 5);
         $products = $this->productRepository->findAll();
```

Comment:
Can we just fetch products available in orders channel?

Reply:
We could if we write a custom repository method for that 👌 As we have a collection of channels not a single channel on the product. But it, of course, could be improved 👍